pkg/slayers/scion_spec.gobra

```diff
@@ -371,8 +423,12 @@ func (s *SCION) EqAbsHeader(ub []byte) bool {
 			&ub[low:high][k] == &ub[low + k]) in
 		let _ := Asserting(forall k int :: {&ub[low:high][:scion.MetaLen][k]} 0 <= k && k < scion.MetaLen ==>
 			&ub[low:high][:scion.MetaLen][k] == &ub[low:high][k]) in
-		s.Path.(*scion.Raw).Base.GetMetaHdr() ==
-		scion.DecodedFrom(binary.BigEndian.Uint32(ub[low:high][:scion.MetaLen]))
+		let metaHdr := scion.DecodedFrom(binary.BigEndian.Uint32(ub[low:high][:scion.MetaLen])) in
+		let seg1 := int(metaHdr.SegLen[0])       in
+		let seg2 := int(metaHdr.SegLen[1])       in
+		let seg3 := int(metaHdr.SegLen[2])       in
+		s.Path.(*scion.Raw).Base.GetBase() ==
+		scion.Base{metaHdr, scion.NumInfoFields(seg1, seg2, seg3), seg1 + seg2 + seg3}
```

Comment:
We have in `raw_spec.gobra`:
```
ghost
requires MetaLen <= len(raw)
requires acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), R56)
decreases
pure func RawBytesToBase(raw []byte) Base
```
Unfortunately this requires a different predicate. Otherwise, I have not found any function similar to that.